### PR TITLE
Tickets/sitcom 797

### PR DESCRIPTION
**Pull request by boutigny**

### CODE REVIEW

File: notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb [hunk 1]
```diff
@@ -0,0 +1,1401 @@
+{
```
Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>First comment would be about renaming file to follow the structure suggested in the last meeting (e.g. sitcom-797_ics_&nbsp;slewing_analysis_positioning.ipynb). Also, the title of the notebook can be more informative, e.g. "SITCOM-797 Slewing Analysis: Positioning" or something else.</p><p> </p><p>I believe that some more context will be given in the technote, so, I think being concise here is fine.</p><p> </p><p>One more general comment is regarding the notebook structure. I think you can create subsections to separate the different analysis you are doing (diff between positions, diff vs elevation/azimuth, etc). In my opinion, it makes reader's life easier.</p><p> </p><p>One last thing. I think it would be nice to add a few comments on results of each individual analysis, instead of only having a final conclusion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/77/discussion/#comment-1387064379'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='0'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>You might be aware of it, but just reminding to use the "lab_black" extension for formatting cells ("%load_ext lab_black).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/77/discussion/#comment-1387064382'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>For such a small list (3 elements), I would prefer to have them shown explicitly like ['xPosition', 'yPostion', 'zPosition'] and so on.</p><p> </p><p>This is just a personal preference, so no need to consider if you don't want to.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/77/discussion/#comment-1387064387'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='5'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>I think you should use subsections to separate the diffente analysis. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/77/discussion/#comment-1387064388'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='6'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>When you start to compute values in line 42 (and onwards), it was a bit hard to follow how you created the results list with the quantities defined in the data frame (line 6). The hardest part to follow was actually the "vect = [ hue list ]" on line 71. Maybe breaking this in some functions? Or simply adding more comments? I don't now, but it was a bit hard to see exactly what is happening.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/77/discussion/#comment-1387064392'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='7'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>Please, add label to the vertical limit lines. if the reader go directly to see the figure, it is not clear what the lines are.</p><p> </p><p>Also, the title of the figure is too general. I think it should be something more meaningful about the specici analysis done here, instead of something more general related to the name of the ticket itsled.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/77/discussion/#comment-1387064395'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>Really nice plots. I would just suggest again to use a more specific title and labels for line limits (actually here they more optional, as they will have same meaning as figure above).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/77/discussion/#comment-1387064398'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='10'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>Have you tried to see what happen with the positioning when doing only a azimuth slew for a fixed elevation? In principle it should be easier to keep the position in that case, but have you tried that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/77/discussion/#comment-1387064401'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (boutigny) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>Thanks for the useful suggestions.</p><ul><li>Notebook renamed as suggested</li><li>More explicit titles</li><li>Create clearer sub-sections</li></ul>

Comment (boutigny) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>Module %load_ext lab_black added</p>

Comment (boutigny) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>I received an opposite comment in the review of another notebook ;-)  But I agree with you and I think that writing the list elements explicitly is clearer --&gt; done </p>

Comment (boutigny) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>Agree. I change the title of this section to clearly explain that I am computing intermediate value that will be used later in the analysis</p>

Comment (boutigny) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>I agree, the logic is difficult to follow. </p><p>The cell has been re-arranged, thanks to lab_black and I have added comments. I hope that it is clearer.</p>

Comment (boutigny) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>I think that Craig did this analysis</p>

Comment (boutigny) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>Titles have been improved</p><p> </p>

Comment (boutigny) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb]: <p>Done</p>

**Review by dsanmartim — APPROVED** [review]
Thanks for addressing my comments. Approved!
